flake.nix

```diff
@@ -84,10 +260,45 @@
                 }
                 # ifstate.nix
                 {
-                  modules = [ ifstate.nixosModules.ifstate ];
+                  modules = [ ifstate.nixosModules.default ];
                   name = "IfState.nix";
                   urlPrefix = "https://codeberg.org/m4rc3l/ifstate.nix/src/branch/main/";
                 }
+                # ifdyndnsd
+                {
+                  modules = [
+                    ifdyndnsd.nixosModule
+                    { _module.args = { inherit pkgs; }; }
+                  ];
+                  name = "ifdyndnsd";
+                  urlPrefix = "https://github.com/astro/ifdyndnsd/blob/main/";
+                }
+                # ixp-manager.nix
```

Comment:
and this

Reply:
I would say not - it is a management software for IXP's and It's not even from us - is just a fork and I am also not maintaining it very well - an update is still pending...